src/jvm/main/org/jetbrains/kotlinx/lincheck/util/Utils.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ * Lincheck
+ *
+ * Copyright (C) 2019 - 2024 JetBrains s.r.o.
+ *
+ * This Source Code Form is subject to the terms of the
+ * Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed
+ * with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.jetbrains.kotlinx.lincheck.util
```

Comment:
Do we need to have to `Utils.kt` files?

Reply:
The idea is to eventually leave only one file, inside the `util` subpackage, and move the stuff from the top-level package `Utils.kt` file either into `util` subpackage or to other more suitable places.

However, I didn't want to clutter this PR with unnecessary changes, thus I haven't modified the top-level `Utils.kt` file.

Reply:
Renamed to `Ensure.kt`